docs/Containers/Home-Assistant.md

```diff
@@ -63,23 +63,40 @@ $ docker-compose up -d
 
 ## <a name="usingBluetooth"></a>Using bluetooth from the container
 
-In order to be able to use BT & BLE devices from HA integrations, make sure that bluetooth is enabled and powered on at the start of the (Rpi) host by editing `/etc/bluetooth/main.conf`:
+In order to be able to use BT & BLE devices from HA integrations, make sure that Bluetooth is enabled:
+
+``` { .console linenums="1" }
```

Comment:
I don't think line-numbers are useful here. Quite the opposite. They are confusing as having numbered lines implies it's a file and not a free-flowing bash-shell.

ps. Is there a reason you don't use the syntax recommended by the Material docs: 
![image](https://user-images.githubusercontent.com/95980324/164173187-c735cbbc-e7dd-49fa-adf3-426040c311f2.png)